src/elements/hx-toggle/index.spec.js

```diff
@@ -0,0 +1,46 @@
+import { fixture, expect } from '@open-wc/testing';
```

Comment:
Please add tests to for the new Toggle Component API to `<hx-toggle-control>` and `<hx-toggle>`.

Reply:
Changes have been done, please review.